index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    |<header id="header">
+        <nav id="nav-bar">
+          <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="logo" class="logo">
+          <ul>
+            <li class="nav-link" >Features</li>
+            <li class="nav-link" >How It Works</li>
+            <li class="nav-link" >Pricing</li>
+          </ul>
+        </nav>
+      </header>
+    
+      <div id="container">
+        <section class="message-box">
+          <h1>Handcrafted, home-made masterpieces</h1>
+          <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input id="email" type="email" name="email" placeholder="example@mail.com" />
+          <input id="submit" type="submit" value="Get Started" class="btn"/>
+          </form>
+        </section>
+    
+        <section id="features">
+          <div class="feature">
+            <i class="fab fa-free-code-camp fa-3x"></i>
+            <div>
+              <h2>Premium Materials</h2>
+              <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+                purchase.</p>
+            </div>
+          </div>
+          <div class="feature">
+            <i class="fas fa-shipping-fast fa-3x"></i>
+            <div>
+              <h2>Fast Shipping</h2>
+              <p>We make sure you receive your trombone as soon as we have finished making it. We also provide free returns
+                if you are not satisfied.</p>
+            </div>
+          </div>
+          <div class="feature">
+            <i class="fas fa-battery-full fa-3x"></i>
+            <div>
+              <h2>Quality Assurance</h2>
+              <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+                pitch of your instrument.</p>
+            </div>
+          </div>
+        </section>
+        <section id="how-it-work">
+          <iframe id="video" src="https://www.youtube.com/embed/JEwOy5Fm1yw"></iframe>
+        </section>
```

Comment:
Please add a header(h2-h6) in this section. A section lacking a heading is not a good HTML coding practice.